style.css

```diff
@@ -6,11 +6,24 @@
   padding: 0;
 }
 
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
 body {
```

Comment:
- [ ] [Optional] All page elements should have similar spacing, borders, shadows, and paddings as the original design, please consider modifying the margin/ padding on your pages so they look as close as possible to the provided design

![Screenshot_20220518_195957](https://user-images.githubusercontent.com/25227022/169135759-05fb199b-47b0-4157-b6c8-6068d59b6b08.png)